api_contracts/contracts/search/AdvancedSearch.md

```diff
@@ -71,8 +115,169 @@ This endpoint returns a list of aggregated award amounts grouped by time period
             + `month`
         + results (array[TimeResult], fixed-type)
 
+## Download Count [/api/v2/download/count/]
+
+Returns the number of transactions that would be included in a download request for the given filter set.
+
+### Download Count Data [POST]
++ Request (application/json)
+    + Attributes (object)
+        + filters: (required, object)
+            + time_period (array[FilterObject])
```

Comment:
This should just be `+ filters (required, FilterObject)` (time_period is included in the FilterObject)